components/Form/index.js

```diff
@@ -78,7 +97,21 @@ export default function Form({ onSubmit, defaultData, formName }) {
     <>
       <form onSubmit={handleSubmit} formName={formName}>
         <StyledHeadingandFoto>
-          {/* NAME INPUT */}
+          <PositioningContainer>
+            <BackLink href={"/contacts"}>{backSVG}</BackLink>
+            {/*-------------------------------- PHOTO INPUT  */}
+            <StyledPhotoLabel htmlFor="photo">
+              Kontact Foto auswählen:
```

Comment:
a small typo, but you can quickly correct it. "Kontakt"

Reply:
thanks!